packages/main/src/themes/sap_horizon_dark/RadioButton-parameters.css

```diff
@@ -1,9 +1,16 @@
 @import "../base/RadioButton-parameters.css";
 
 :root {
-	--_ui5_radio_button_border_width: 0.125rem;
-	--_ui5_radio_button_border_radius: 0.75rem;
+	--_ui5_radio_button_border_width: var(--sapContent_FocusWidth);
+	--_ui5_radio_button_border_radius: 0.5rem;
 	--_ui5_radio_button_label_color: var(--sapField_TextColor);
-	--_ui5_radio_button_border_readonly_focus_style: dashed;
-	--_ui5_radio_button_focus_dist: 0.375rem;
+	--_ui5_radio_button_inner_ring_radius: 27.5%;
+	/* --_ui5_radio_button_outer_ring_hover_shadow: 0 0 0.125rem fade(var(--sapNeutralColor), 72);  does not work??? */
+	--_ui5_radio_button_outer_ring_hover_shadow: 0 0 0.125rem rgba(234, 236, 238, 0.72); /* need of parameter */
+	--_ui5_radio_button_outer_ring_hover_shadow_error: 0 0 0.125rem rgba(250, 97, 97, 0.72); /* need of parameter*/
+	--_ui5_radio_button_outer_ring_hover_shadow_warning: 0 0 0.125rem rgba(255, 223, 114, 0.72); /* need of parameter*/
+	--_ui5_radio_button_outer_ring_hover_shadow_success: 0 0 0.125rem rgba(151, 221, 64, 0.72); /* need of parameter*/
+	--_ui5_radio_button_outer_ring_hover_shadow_information: 0 0 0.125rem rgba(77, 177, 255, 0.72); /* need of parameter*/
```

Comment:
You can use the following parameters: 
  --sapContent_Interaction_Shadow;
  --sapContent_Selected_Shadow;
  --sapContent_Negative_Shadow;
  --sapContent_Critical_Shadow;
  --sapContent_Positive_Shadow;
  --sapContent_Informative_Shadow;

Available in [theming-base-content](https://github.com/SAP/theming-base-content/blob/master/content/Base/baseLib/sap_horizon/css_variables.css#L138).